openshift/f8a-notification.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1     
+kind: Template
+labels:
+  template: f8a-notification
+metadata:
+  name: f8a-notification
+  annotations:
+    description: f8a-notification
+objects:
+- apiVersion:  batch/v2alpha1
+  kind: CronJob
+  metadata:
+    name: f8a-notification-scheduler
```

Comment:
Let's be consistent here. `f8a-notification` or `f8a-notification-scheduler`? :)